pom.xml

```diff
@@ -137,7 +137,7 @@
     <!-- Version used for internal directory structure -->
     <hive.version.short>2.3</hive.version.short>
     <!-- note that this should be compatible with Kafka brokers version 0.10 and up -->
-    <kafka.version>3.3.2</kafka.version>
+    <kafka.version>3.4.1</kafka.version>
```

Comment:
My bad. I thought you suggested to upgrade from 3.4.0 to 3.4.1. 
We don't upgrade dependency in release branches when there is a breaking code change.